source/patterns/00-atoms/block/image/image.twig

```diff
@@ -6,6 +6,8 @@
     <source {{- html.attributes(source) }}>
   {%- endfor -%}
 
+  {%- set image = image|default({})|merge({class: image.class|default([])|merge(['image']) }) -%}
```

Comment:
I'd favour putting all the styles on the `<picture>` itself.

Reply:
Current styles do not work on picture. Will play around to see if I can make it work